scripts/caldp-docker-run-container

```diff
@@ -11,11 +11,11 @@
 
 # Choose the CALDP docker image configured by an installed
 # caldp-image-config
-source caldp-image-config latest
```

Comment:
is this a bug? 
`caldp-docker-run-pipeline j8cb010b0 file:inputs file:outputs
/Users/bhayden/miniconda2/envs/caldp/bin/caldp-docker-run-container: line 14: CALDP_IMAGE_TAG: unbound variable`

Reply:
I think it's necessary to remove that so that developers can choose between stable and latest by defining  them in the config file as needed and sourcing it externally.  So there should probably be a README comment about sourcing the config file?  Is that sufficient?

Reply:
On second thought,  we should keep it,  I just changed caldp-image-config wrong.

My proposal is this:  to solve the problem of defining multiple image tags,  you need to edit caldp-image-config if you want to change  defaults,   then install it (which you have to do anyway).  You dont need to source it.

Reply:
It's weird because the CALDP_IMAGE_TAG is set by caldp-image-config, using the tag that's passed when calling the script. So this bit is circular. You need a tag in order to pass a tag into caldp-image-config, in order to get a tag back.

We could set CALDP_IMAGE_TAG as a required ENV variable for the user, since it's really needed for both building the container and then any later references to it.

Reply:
oops, looks like we commented at about the same time.

Basically you're saying we have latest be the default in caldp-image-config, then only change the default in that one place if we want a different tag?

Reply:
Yeah,  that's basically it.    To be really clear if I wasn't,  the tag should no longer be an override parameter to caldp-image-config,  the env var should just be set directly:  export CALDP_IMAGE_TAG="latest".  That will define it  in the env.   Then caldp-image-config  is automatically sourced everywhere but with no parameters...  and  we can add "source caldp-image-config" back to caldp-docker-run-container.   (The reason I dropped the entire line was that just removing "latest" wasn't sufficient,  caldp-image-config also has  to be modified so that the tag is no longer a parameter,  the default doesn't work correctly.)